collection/entry.rkt

```diff
@@ -87,11 +87,13 @@
 
 
 (define (mapping-keys key-function)
-  (mapping (λ (e) (match e [(entry k v) (entry (key-function k) v)]))))
+  (mapping (λ (e) (match-define (entry k v) e)
+                  (entry (key-function k) v))))
```

Comment:
Looks like using `~focus-replacement-on` when there's only one form in the context is a bad idea. This formatting is terrible.